src/talk/talk.window.js

```diff
@@ -65,6 +67,8 @@ function createTalkWindow() {
 		window.show()
 	})
 
+	applyContextMenu(window)
```

Comment:
Should we also apply the context menu to the authentication window? For example, to paste the server url by the context menu.